lib/utilities/monitoringHandler.js

```diff
@@ -167,23 +177,23 @@ function routeHandler(req, res, log, cb) {
  * @param {RequestLogger} log - werelogs logger instance
  * @return {void}
  */
-function monitoringHandler(clientIP, req, res, log) {
+async function monitoringHandler(clientIP, req, res, log) {
```

Comment:
If we don't await any of the returned/called function, why do we want this function to be async?

Reply:
it calls `routeHandler()`, which is async because it calls/awaits' promClient's `metrics()` function

Reply:
Unless you are awaiting an async function, you should be able to call async function from sync ones, it will behave as expected :smile: